dataset/companies/FNB.yml

```diff
@@ -0,0 +1,8 @@
+ - Name: "McDonalds"
+    Description: ""
+    Website: "https://www.mcdonalds.com/"
+    alternatives: []
```

Comment:
Please capitalize alternatives. So it will be `Alternatives`.

I have pushed a change to this repo to make it capitalized